src/ranking_collecter/ranking_collector.py

```diff
@@ -0,0 +1,26 @@
+from string import Template
+import requests
+import user_info
+
+def collect_ranks(event, context):
+    usernames = user_info.USERNAMES
+    weekly_rankings = []
```

Comment:
nitpick, generally not good practice to lump this with other dynamically loaded variables, declare it when needed